source/_UIAHandler.py

```diff
@@ -59,13 +59,15 @@
 	"SysListView32",
 	"EXCEL7",
 	"Button",
-	# #7497: Windows 10 Fall Creators Update has an incomplete UIA implementation for console windows, therefore for now we should ignore it.
-	# It does not implement caret/selection, and probably has no new text events.
-	"ConsoleWindowClass",
 	# #8944: The Foxit UIA implementation is incomplete and should not be used for now.
 	"FoxitDocWnd",
 ]
 
+# #7497: Windows 10 Fall Creators Update has an incomplete UIA implementation for console windows, therefore for now we should ignore it.
+# It does not implement caret/selection, and probably has no new text events.
+if not config.conf['UIA']['consoleUIA']:

```

Comment:
I assume you will extend this check in the future? On windows 7, 8 and 10 below fall creators update, this will break in a major way probably.

Reply:
Yes – the final PR for this work will introduce auto-detection for UIA/legacy console support which will make checking more robust as a consequence.